Clean cmake failing (visual studio 2017)

As of the latest commit in master (haven't tried earlier ones)

CMake Error at W:/Program Files (x86)/Microsoft Visual Studio/2017/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.12/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find PostgreSQL (missing: PostgreSQL_LIBRARY
  PostgreSQL_INCLUDE_DIR PostgreSQL_TYPE_INCLUDE_DIR)		W:/Program Files (x86)/Microsoft Visual Studio/2017/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.12/Modules/FindPackageHandleStandardArgs.cmake	137	


Took a look in taopq-master\taopq-master\cmake\taopqConfig.cmake.in

and it seems like it specifically needs PostgreSQL 9.6.9. Is that the case? I've got postgresql 12 installed.

Either way, figured I'd try the conan route, and the package isn't found in the remote.

taocpp-taopq/20200215: Not found in local cache, looking in remotes...
taocpp-taopq/20200215: Trying with 'conan-center'...
ERROR: Unable to find 'taocpp-taopq/20200215' in remotes

I've got the libpq libraries built, but not sure how to proceed,

--- a/cmake/taopqConfig.cmake.in
+++ b/cmake/taopqConfig.cmake.in
@@ -3,7 +3,7 @@ include(CMakeFindDependencyMacro)
 
 list(APPEND CMAKE_MODULE_PATH ${taopq_CMAKE_DIR})
 
-find_package(PostgreSQL 9.6.9 REQUIRED MODULE)
+find_package(PostgreSQL REQUIRED MODULE)
 list(REMOVE_AT CMAKE_MODULE_PATH -1)
 
 if(NOT TARGET taocpp::taopq)

